CHANGELOG.md

```diff
@@ -1,4 +1,17 @@
 ## 1.1.6 (Unreleased)
```

Comment:
Does the release date need to be entered or does that happen later?

Reply:
That  gets updated by the Terraform team during their release process.